### DISCUSSION TITLE
scripts/utils/show_translation_result.sh file is missing in MT scoring

### DISCUSSION THREAD

**Discussion by pratikkumar018** [espnet/espnet#4241] [General]
I am trying to run the recipe [iwslt14/mt1](https://github.com/espnet/espnet/tree/cb8181a99bb59f8444b59a36c38878f95570faaf/egs2/iwslt14/mt1). But in scoring section line no. 1281         scripts/utils/show_translation_result.sh --case $tgt_case "${mt_exp}" > "${mt_exp}"/RESULTS.md
it is failing saying that scripts/utils/show_translation_result.sh file is missing. Please look into this,

**Comment by sw005320** [comment]
@pyf98, could you deal with it?
I also cannot find it

**Comment by pyf98** [comment]
Hi, thanks for the question. Yes, I think this script has not been completely implemented yet. But it does not affect the decoding and scoring. So you can find the results in the corresponding folder.

In https://github.com/espnet/espnet/pull/4149, there is an initial version of `show_translation_result.sh` by Brian, which can show some of the decoding results.

I had a more complete version several days ago, but it was based on a separate python script to extract the results from a txt file. I will check it and maybe upload it when I have time.

**Reply by pratikkumar018** [reply]
Thanks for the info.

**Comment by pyf98** [comment]
Hi, I've updated the `show_translation_result.sh` in this PR: https://github.com/espnet/espnet/pull/4330
